services/web/server/src/simcore_service_webserver/computation_subscribe.py

```diff
@@ -60,19 +57,23 @@ async def progress_message_parser(app: web.Application, data: bytes) -> None:
                 }
             ]
             await send_messages(app, f"{rabbit_message.user_id}", messages)
+            return True
     except ProjectNotFoundError:
         log.warning(
             "project related to received rabbitMQ progress message not found: '%s'",
```

Comment:
this can happen when:
- someone fiddles with the database while running a project
- it can potentially happen ```NodeNotFoundError``` if someone deletes a node while running a computation (this is not allowed anymore but was at some point)
- I guess we could look for this but again, I would maybe put some identifier as well. we can discuss this.